Code/ForceField/MMFF/Params.cpp

```diff
@@ -2157,14 +2157,14 @@ MMFFCovRadPauEleCollection *MMFFCovRadPauEleCollection::getMMFFCovRadPauEle(
 
 MMFFCovRadPauEleCollection::MMFFCovRadPauEleCollection(
     std::string mmffCovRadPauEle) {
-  if (mmffCovRadPauEle == "") {
+  if (mmffCovRadPauEle.empty()) {
     mmffCovRadPauEle = defaultMMFFCovRadPauEle;
   }
   std::istringstream inStream(mmffCovRadPauEle);
   std::string inLine = RDKit::getLine(inStream);
   while (!(inStream.eof())) {
     if (inLine[0] != '*') {
-      MMFFCovRadPauEle mmffCovRadPauEleObj;
+      MMFFCovRadPauEle mmffCovRadPauEleObj{};
```

Comment:
It's not clear to me why this construct is an improvement over what was there before.

Reply:
clang-tidy suggested those. I didn't mean to include them, they must have slipped in.